scopes/component/component/component.ui.runtime.tsx

```diff
@@ -55,28 +55,30 @@ export class ComponentUI {
    */
   private activeComponent?: ComponentModel;
 
+  private copyNpmId = () => {
+    const packageName = this.activeComponent?.packageName;
+    if (packageName) {
+      const version = this.activeComponent?.id.version;
+      const versionString = version ? `@${version}` : '';
+      copy(`${packageName}${versionString}`);
+    }
+  };
+
   /**
    * key bindings used by component aspect
    */
   private keyBindings: CommandEntry[] = [
     {
-      id: 'copyBitId', // extract to constant!
+      id: 'copyBitId', // TODO - extract to a component!
```

Comment:
1. prefix with `'component.'`
1. extract to constant! you can move it to another file in this same component